### PR TITLE
[docs] add JS examples for double texting and human in the loop for cloud

### PR DESCRIPTION
**Pull request by isahers1**

### CODE REVIEW

File: docs/docs/cloud/how-tos/enqueue_concurrent.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    client = get_client(url="whatever-your-deployment-url-is")
+    assistant_id = "agent"
+    thread = await client.threads.create()
+    ```
+
+=== "Javascript"
+
+    ```js
+    import { Client } from "@langchain/langgraph-sdk";
+    import { coerceMessageLikeToMessage } from "@langchain/core/messages"
```
Comment (jacoblee93) [inline docs/docs/cloud/how-tos/enqueue_concurrent.md:28]: Can we avoid using this?

File: docs/docs/cloud/how-tos/enqueue_concurrent.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+=== "Javascript"
+
+    ```js
+    const first_run = await client.runs.create(
+        thread["thread_id"],
+        assistant_id,
+        input={"messages": [{"role": "human", "content": "what's the weather in sf?"}]},
+    )
+
+    const second_run = await client.runs.create(
```
Comment (jacoblee93) [inline docs/docs/cloud/how-tos/enqueue_concurrent.md:62]: Camel case

File: docs/docs/cloud/how-tos/enqueue_concurrent.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+=== "Javascript"
+
+    ```js
+    await client.runs.join(thread["thread_id"], second_run["run_id"]);
+
+    const state = await client.threads.getState(thread["thread_id"]);
+
+    const baseMessages = state["values"]["messages"].map((message) =>
+        coerceMessageLikeToMessage(message);
```
Comment (jacoblee93) [inline docs/docs/cloud/how-tos/enqueue_concurrent.md:92]: Convention is two space tabs

File: docs/docs/cloud/how-tos/enqueue_concurrent.md (hunk 1 above)
Comment (vbarda) [reply docs/docs/cloud/how-tos/enqueue_concurrent.md:28]: +1

File: docs/docs/cloud/how-tos/reject_concurrent.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+    client = get_client(url="whatever-your-deployment-url-is")
+    assistant_id = "agent"
+    thread = await client.threads.create()
+    ```
+
+=== "Javascript"
+
+    ```js
+    import { Client } from "@langchain/langgraph-sdk";
+    import { coerceMessageLikeToMessage } from "@langchain/core/messages"
```
Comment (vbarda) [inline docs/docs/cloud/how-tos/reject_concurrent.md:25]: let's probably remove

File: docs/docs/cloud/how-tos/interrupt_concurrent.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+    ```
+
+=== "Javascript"
+
+    ```js
+    import { Client } from "@langchain/langgraph-sdk";
+    import { coerceMessageLikeToMessage } from "@langchain/core/messages"
+
+    const client = new Client({apiUrl:"whatever-your-deployment-url-is"});
+    const assistant_id = "agent";
```
Comment (vbarda) [inline docs/docs/cloud/how-tos/interrupt_concurrent.md:30]: camelcase

File: docs/docs/cloud/how-tos/enqueue_concurrent.md [hunk 6]
```diff
@@ -1,11 +1,29 @@
...
-First, let's import our required packages and instantiate our client, assistant, and thread.
 
+First, we will define a quick helper function for printing out JS model outputs (you can skip this if using Python):
+
+```js
+import { coerceMessageLikeToMessage } from "@langchain/core/messages"
+
+function prettyPrint(m) {
+  m = coerceMessageLikeToMessage(m);
+  let padded = " " + m._getType() + " ";
```
Comment (vbarda) [inline docs/docs/cloud/how-tos/enqueue_concurrent.md:15]: i still think we don't need `coerceMessageLikeToMessage` only to use it for `m._getType`, can't we just do `m["type"]` for this? or are those two different?

File: docs/docs/cloud/how-tos/enqueue_concurrent.md [hunk 7]
```diff
@@ -1,11 +1,29 @@
...
+  let sepLen = Math.floor((80 - padded.length) / 2);
```
Comment (vbarda) [inline docs/docs/cloud/how-tos/enqueue_concurrent.md:16]: let -> const for all of these

File: docs/docs/cloud/how-tos/enqueue_concurrent.md [hunk 8]
```diff
@@ -84,23 +102,12 @@ Verify that the thread has data from both runs:
...
-        let padded = " " + m._getType() + " ";
-        let sepLen = Math.floor((80 - padded.length) / 2);
-        let sep = "=".repeat(sepLen);
-        let secondSep = sep + (padded.length % 2 ? "=" : "");
-        
-        console.log(`${sep}${padded}${secondSep}`);
-        console.log("\n\n");
-        console.log(m.content);
+    for (const m of state["values"]["messages"]) {
+      prettyPrint(m);
```
Comment (vbarda) [inline docs/docs/cloud/how-tos/enqueue_concurrent.md:110]: nice!

File: docs/docs/cloud/how-tos/human_in_the_loop_breakpoint.md [hunk 9]
```diff
@@ -34,9 +32,7 @@ In this how-to we use a simple ReAct style hosted graph (you can see the full co
     import { Client } from "@langchain/langgraph-sdk";
 
     const client = new Client({apiUrl:"whatever-your-deployment-url-is"});
-    let assistants = await client.assistants.search();
-    assistants = assistants.filter(a => !a.config);
-    const assistant = assistants[0];
+    let assistantId = "agent"
```
Comment (vbarda) [inline docs/docs/cloud/how-tos/human_in_the_loop_breakpoint.md:35]: let -> const

File: docs/docs/cloud/how-tos/human_in_the_loop_edit_state.md [hunk 10]
```diff
@@ -30,9 +28,7 @@ First, we need to setup our client so that we can communicate with our hosted gr
     import { Client } from "@langchain/langgraph-sdk";
 
     const client = new Client({apiUrl:"whatever-your-deployment-url-is"});
-    let assistants = await client.assistants.search();
-    assistants = assistants.filter(a => !a.config);
-    const assistant = assistants[0];
+    let assistantId = "agent";
```
Comment (vbarda) [inline docs/docs/cloud/how-tos/human_in_the_loop_edit_state.md:31]: let -> const

File: docs/docs/cloud/how-tos/human_in_the_loop_time_travel.md [hunk 11]
```diff
@@ -27,9 +25,7 @@ First, we need to setup our client so that we can communicate with our hosted gr
     import { Client } from "@langchain/langgraph-sdk";
 
     const client = new Client({apiUrl:"whatever-your-deployment-url-is"});
-    let assistants = await client.assistants.search();
-    assistants = assistants.filter(a => !a.config);
-    const assistant = assistants[0];
+    let assistantId = agent;
```
Comment (vbarda) [inline docs/docs/cloud/how-tos/human_in_the_loop_time_travel.md:28]: let-> const

File: docs/docs/cloud/how-tos/human_in_the_loop_time_travel.md [hunk 12]
```diff
@@ -124,31 +120,31 @@ To rerun from a state, we need to pass in the `checkpoint_id` into the config of
...
-            input: null,
-            streamMode: "updates",
-            config: {"configurable":{"thread_ts":state_to_replay['checkpoint_id']}},
-        }
+      thread["thread_id"],
+      assistantId,
+      {
+        input: null,
+        streamMode: "updates",
+        config: {"configurable":{"thread_ts":stateToReplay['checkpoint_id']}},
```
Comment (vbarda) [inline docs/docs/cloud/how-tos/human_in_the_loop_time_travel.md:141]: nit: missing whitespaces in config dictionary here

File: docs/docs/cloud/quick_start.md [hunk 13]
```diff
@@ -224,7 +224,7 @@ async for chunk in client.runs.stream(
         input=input,
         stream_mode="updates",
     ):
-    if chunk.data and "run_id" not in chunk.data:
+    if chunk.data and chunk.event != "metadata": 
             print(chunk.data)
```
Comment (vbarda) [inline docs/docs/cloud/quick_start.md:228]: this looks like a suspiciously long indent

File: docs/docs/cloud/how-tos/enqueue_concurrent.md [hunk 14]
```diff
@@ -0,0 +1,184 @@
...
+This guide assumes knowledge of what double-texting is, which you can learn about in the [double-texting conceptual guide](https://langchain-ai.github.io/langgraph/cloud/concepts/#double-texting).
+
+The guide covers the `enqueue` option for double texting, which adds the interruptions to a queue and executes them in the order they are received by the client. Below is a quick example of using the `enqueue` option.
+
+
+First, we will define a quick helper function for printing out JS model outputs (you can skip this if using Python):
+
+```js
+function prettyPrint(m) {
+  m = coerceMessageLikeToMessage(m);
```
Comment (vbarda) [inline docs/docs/cloud/how-tos/enqueue_concurrent.md:12]: remove

File: docs/docs/cloud/how-tos/enqueue_concurrent.md [hunk 15]
```diff
@@ -0,0 +1,184 @@
...
+    thread = await client.threads.create()
+    ```
+
+=== "Javascript"
+
+    ```js
+    import { Client } from "@langchain/langgraph-sdk";
+    
+
+    const client = new Client({apiUrl:"whatever-your-deployment-url-is"});
```
Comment (vbarda) [inline docs/docs/cloud/how-tos/enqueue_concurrent.md:46]: nit -- add spaces like `Client({ apiUrl:"whatever-your-deployment-url-is" });`

File: docs/docs/cloud/how-tos/human_in_the_loop_breakpoint.md [hunk 16]
```diff
@@ -0,0 +1,110 @@
...
+    assistant_id = "agent"
+    thread = await client.threads.create()
+    ```
+
+=== "Javascript"
+
+    ```js
+    import { Client } from "@langchain/langgraph-sdk";
+
+    const client = new Client({apiUrl:"whatever-your-deployment-url-is"});
```
Comment (vbarda) [inline docs/docs/cloud/how-tos/human_in_the_loop_breakpoint.md:34]: nit spaces in the dict here

File: docs/docs/cloud/how-tos/human_in_the_loop_edit_state.md [hunk 17]
```diff
@@ -0,0 +1,189 @@
...
+        interrupt_before=["action"],
+    ):
+        if chunk.data and chunk.event != "metadata": 
+            print(chunk.data)
+    ```
+
+=== "Javascript"
+
+    ```js
+    const input = {
```
Comment (vbarda) [inline docs/docs/cloud/how-tos/human_in_the_loop_edit_state.md:60]: let's make input look similar in both python and js so it's nicer to switch back and forth (i.e. if it's multiline for js, it should be also multiline for python)

File: docs/docs/cloud/how-tos/interrupt_concurrent.md [hunk 18]
```diff
@@ -0,0 +1,177 @@
...
+
+This guide assumes knowledge of what double-texting is, which you can learn about in the [double-texting conceptual guide](https://langchain-ai.github.io/langgraph/cloud/concepts/#double-texting).
+
+The guide covers the `interrupt` option for double texting, which interrupts the prior run of the graph and starts a new one with the double-text. This option does not delete the first run, but rather keeps it in the database but sets its status to `interrupted`. Below is a quick example of using the `interrupt` option.
+
+First, we will define a quick helper function for printing out JS model outputs (you can skip this if using Python):
+
+```js
+function prettyPrint(m) {
+  m = coerceMessageLikeToMessage(m);
```
Comment (vbarda) [inline docs/docs/cloud/how-tos/interrupt_concurrent.md:11]: remove

**Review by vbarda — APPROVED** [review]
lgtm -- small comments and should be good to merge after

**Pull request merged**